blog/2023-06-10-conda-libmamba-solver-rollout.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: "conda & conda-libmamba-solver: Roll-out plan 2023"
+slug: "2023-06-10-conda-libmamba-solver-rollout"
+description: |
+    Learn about the upcoming rollout plan for the conda-libmamba-solver, bringing enhanced performance and user experience to conda.
+authors: [jezdez]
+tags: [conda, mamba, solver, libmamba, dependency solver, performance, user experience, conda-forge, pycosat, plugin package, Anaconda Distribution, miniconda, CEP 8, CEP 9]
+
+---
+## Background
+
+
+Over a year ago the conda team at Anaconda, together with conda developers at Quansight and QuantStack, [announced the release](https://www.anaconda.com/blog/a-faster-conda-for-a-growing-community) of a new solver for conda based on the solver that the [mamba project](https://mamba.readthedocs.io/) had developed on top of the venerable libsolv project.
+
+The dependency solver is one of the main pillars that make up the conda package and environment manager (next to network & IO stack and the command line interface integration for example) and has been the source of many performance and user experience issues over the years, given the growth of the conda ecosystem. Users of conda-forge and other larger conda channels in particular struggled with the current solver implementation based on pycosat.
+
+[Fast-forward to December 2022](https://www.anaconda.com/blog/conda-is-fast-now), we’ve encouraged the use of the libmamba solver via the separate plugin package “conda-libmamba-solver" as a way to trial the use of the solver and get more feedback from real-world users. In case you haven't had the chance to provide your feedback, please head over to the [conda-libmamba-solver repository on GitHub](https://github.com/conda/conda-libmamba-solver).
```

Comment:
```suggestion
[Fast-forward to December 2022](https://www.anaconda.com/blog/conda-is-fast-now); we’ve encouraged the use of the libmamba solver via the separate plugin package “conda-libmamba-solver" as a way to trial the use of the solver and get more feedback from real-world users. In case you haven't had the chance to provide your feedback, please head over to the [conda-libmamba-solver repository on GitHub](https://github.com/conda/conda-libmamba-solver).
```